.gitignore

```diff
@@ -10,3 +10,4 @@ compiled
 .rpt2_cache
 docs
 build
+package-lock.json
```

Comment:
Can you remove the changed `.gitignore` from the PR?

Reply:
sure...is there a problem with it? I notice that you dont track package-lock in this repo. I found myself doing `git add .` which inevitably brought in package-lock.json

Reply:
Oh haha you're right! Actually, let's keep your change. We track `yarn.lock`, which means we shouldn't track `package-lock.json`

Reply:
sweet...then we are good to go